### ISSUE TITLE
Scenario names and descriptions from STEX objects broken

### ISSUE THREAD

**Issue by AaronVanGeffen** [OpenRCT2/OpenRCT2#7509]
**OS:** Linux, macOS
**Version:** 0.1.3
**Commit/Build:** c7b40cec7cf55326aa4b33eeb1d71b58311660ef

Loading scenario names and descriptions from STEX objects is no longer working.

Doing a bisect reveals this to have been caused by the merger of the `json-objects` branch. I can't get every every commit in the branch to compile, but 06da0b2fb extracts from the STEX objects just fine, whereas 0ab3d0955 does not.

Broken behaviour on c7b40cec7cf55326aa4b33eeb1d71b58311660ef:
![screenshot_20180511_223342](https://user-images.githubusercontent.com/604665/39945886-62c8b69a-556c-11e8-9213-371aca45600a.png)

Expected behaviour on v0.1.2:
![screenshot_20180511_224047](https://user-images.githubusercontent.com/604665/39945887-62e6937c-556c-11e8-968b-a4a5e36ca669.png)

**Comment by Gymnasiast** [comment]
Also affects German and probably every other language that does not have overrides in the language file.

**Comment by Gymnasiast** [comment]
I have extracted all the STEX descriptions. This means that all default scenarios now have their localised descriptions back. Besides two scenarios I created myself, I don't know of any custom scenario using it in the wild. If one does use it, it will simply ignore any missing STEX file.

STEX support will be removed at some point; our own save format will have a string table as part of the spec, so no external objects will be needed. Also see here: https://github.com/OpenRCT2/OpenRCT2/wiki/STEX-support-removal
